src/gc/gcee.cpp

```diff
@@ -390,6 +390,7 @@ size_t GCHeap::GetNow()
     return hp->get_time_now();
 }
 
+#if defined(GC_PROFILING) //UNIXTODO: Enable this for FEATURE_EVENT_TRACE
 void ProfScanRootsHelper(Object** ppObject, ScanContext *pSC, DWORD dwFlags)
 {
 #if defined(GC_PROFILING) || defined(FEATURE_EVENT_TRACE)
```

Comment:
I would like it to be a cue for whoever is going to separate GC_PROFILING and FEATURE_EVENT_TRACE.

NOTE: This needs to work as we see before my change for FEATURE_EVENT_TRACE, but currently it does not and that it the reason for UNIXTODO, I could have defined it out for linux only but GC_PROFILING is more generic